### DISCUSSION TITLE
Update on the development branch

### DISCUSSION THREAD

**Discussion by kaiyux** [NVIDIA/TensorRT-LLM#2095] [Announcements]
Hi,

The TensorRT-LLM team is pleased to announce that we have pushed an update to the [development](https://github.com/NVIDIA/TensorRT-LLM/tree/main) branch (and the [Triton backend](https://github.com/triton-inference-server/tensorrtllm_backend/tree/main)) this Aug 7th, 2024.

This [update](https://github.com/NVIDIA/TensorRT-LLM/pull/2094) includes:

* Model Support
  - Support GLM4 models, see `examples/chatglm/README.md`.
* API
  - [BREAKING CHANGE] The build time argument `context_fmha_fp32_acc` is moved to runtime for decoder models.
  - Supported Falcon models in the Python high level API.
* Bug fixes
  - Fixed segmentation fault in TopP sampling layer, thanks to the contribution from @akhoroshev in #2039. (#2040)
  - Fixed the failure when converting the checkpoint for Mistral Nemo model. (#1985)
  - Propagate exclude_modules to weight only quantization, thanks to the contribution from @fjosw in #2056.
* Documentation
  - Fixed wrong links in README, thanks to the contribution from @Tayef-Shah in #2028.
  - Fixed some typos in the documentation, thanks to the contribution from @lfz941 in #1939.

Thanks,
The TensorRT-LLM Engineering Team

**Comment by MARD1NO** [comment]
The update link is wrong(skip to pr2053 :D

**Reply by kaiyux** [reply]
Sorry for the mistake, I've fixed it. Thanks a lot for your attention and support!
